Userland/Utilities/base64.cpp

```diff
@@ -10,15 +10,38 @@
 #include <LibCore/System.h>
 #include <LibMain/Main.h>
 
+static ErrorOr<String> wrap_encoded_output(size_t column, StringView encoded)
+{
+    if (column == 0)
+        return String::from_utf8(encoded);
```

Comment:
Sorry for the second review, but I was thinking about this again, and what if we just avoid allocating the resulting string altogether? And just print `column` characters at a time.

Something like:
```c++
static void print_wrapped_output(size_t column, StringView encoded)
{
    while (!encoded.is_empty()) {
        auto segment_length = min(column, encoded.length());

        outln("{}", encoded.substring_view(0, segment_length));
        encoded = encoded.substring_view(segment_length);
    }
}
```

And to call it:
```c++
    if (maybe_column.has_value() && *maybe_column > 0) {
        print_wrapped_output(*maybe_column, encoded);
        return 0;
    }
```

Reply:
This looks like a much better approach; its not like we need to use the wrapped result elsewhere. I used your changes but decided to keep the sanity check in `print_wrapped_output()` as looking at the code it seems an infinite loop can occur. Then again, I'm am pretty sure it will only ever be called within the if branch in main so it may just be unnecessary

Reply:
The only problem with the early return inside the helper is that if the user passes `-w 0`, we now print nothing. So maybe make that be `VERIFY(column > 0)` instead if you want to be safe.